### PR TITLE
last changes

### PR DESCRIPTION
**Pull request by jasminecoder**

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -17,7 +17,8 @@ def index
       @ransack_courses = Course.published.approved.ransack(params[:courses_search], search_key: :courses_search)
       # @courses = @ransack_courses.result.includes(:user)
 
-      @pagy, @courses = pagy(@ransack_courses.result.includes(:user))
+      @pagy, @courses = pagy(@ransack_courses.result.includes(:user, :course_tags, :course_tags => :tag))
```
Comment (johnreitano) [inline app/controllers/courses_controller.rb]: It's better to use the new symbol key notation `course_tags: :tag`

Comment (johnreitano) [reply app/controllers/courses_controller.rb]: and same thing a few times below....

File: app/controllers/courses_controller.rb [hunk 2]
```diff
@@ -97,6 +107,7 @@ def create
         format.html { redirect_to @course, notice: 'Course was successfully created.' }
         format.json { render :show, status: :created, location: @course }
       else
+        @tags = Tag.all.where.not(course_tags_count: 0)
         format.html { render :new }
```
Comment (johnreitano) [inline app/controllers/courses_controller.rb]: I would put the `@tags=` line inside the format.html block.

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -4,13 +4,14 @@
 // that code so it'll be compiled.
 
 import Rails from "@rails/ujs";
-import Turbolinks from "turbolinks";
+// import Turbolinks from "turbolinks";
```
Comment (johnreitano) [inline app/javascript/packs/application.js:7]: remove this line.

**Review by johnreitano — CHANGES_REQUESTED** [review]

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (jasminecoder) [reply app/controllers/courses_controller.rb]: fixed

File: app/controllers/courses_controller.rb (hunk 2 above)
Comment (jasminecoder) [reply app/controllers/courses_controller.rb]: fixed

File: app/javascript/packs/application.js (hunk 3 above)
Comment (jasminecoder) [reply app/javascript/packs/application.js:7]: no

**Review by johnreitano — APPROVED** [review]

**Pull request merged**